packages/forms-web-app/src/controllers/examination/email.js

```diff
@@ -2,24 +2,61 @@ const {
 	routesConfig: {
 		examination: {
 			directory: examinationDirectory,
-			pages: { nameMyself, email }
+			pages: { email }
 		}
 	}
 } = require('../../routes/config');
 
+const config = require('../../config');
+const examinationSessionStorage = config?.sessionStorage?.examination;
+
 const pageData = {
-	backLinkUrl: `${examinationDirectory + nameMyself.route}`,
 	id: email.id,
 	pageTitle: email.name,
-	title: email.name
+	title: email.name,
+	hint: "We'll use your email address to confirm we've received your submission. We will not publish your email address."
 };
 
 const getEmail = async (req, res) => {
+	const requestSession = req.session;
+	const userEmail = requestSession?.[examinationSessionStorage.name]?.userEmail;
+	const currentView = requestSession?.currentView;
```

Comment:
What is current view doing?

Reply:
it passes the name object (agent, organization, onBehalf) from the route config through the forward-view middleware in the name.js (as we have 3 different names using the same controller).

Reply:
knowing which is the right view we can create the proper backlink to it